deployments/Dockerfiles/autonomy/Dockerfile

```diff
@@ -2,25 +2,8 @@ ARG AEA_VERSION=latest
 
 FROM valory/open-aea-user:${AEA_VERSION}
 
+RUN apt install git net-tools sudo -y
 
-WORKDIR /home/ubuntu
-ENV PATH=$PATH:/home/ubuntu/.local/bin
+COPY scripts /root/scripts
 
-RUN sudo apt-get update --fix-missing
-RUN sudo apt-get autoremove
-RUN sudo apt-get autoclean
-RUN sudo apt-get install python3.10 python3.10-dev git cmake autoconf automake build-essential libffi-dev libtool pkg-config -y
-RUN cd /usr/bin && sudo rm python3 && sudo ln -s python3.10 python3 && sudo ln -s python3.10 python
```

Comment:
No need to install the python interpreters and dev libraries, they are included in the base image